### PR TITLE
Remove remains of Travis

### PR DESCRIPTION
**Pull request by Alkarex**
Follow-up of https://github.com/FreshRSS/FreshRSS/pull/3920

### CODE REVIEW

File: docs/en/developers/03_Running_tests.md [hunk 1]
```diff
@@ -27,8 +27,10 @@ make lint-fix # or, to fix the errors detected by the linter
 
 Similarly to PHPUnit, it downloads a [PHP\_CodeSniffer](https://github.com/squizlabs/PHP_CodeSniffer) binary (i.e. `phpcs` or `phpcbf` depending on the command) and verifies its checksum.
 
-## Travis
+## GitHub Actions for Continuous Integration
 
-Tests are automatically run when you open a pull request on GitHub. It is done with [Travis CI](https://travis-ci.org/FreshRSS/FreshRSS/). This is done to ensure there is no regressions in your code. We cannot merge a PR if the tests fail so we'll ask you to fix bugs before to review your code.
+Tests are automatically run when you open a pull request on GitHub.
+They are done with [GitHub Actions](https://github.com/FreshRSS/FreshRSS/actions).
+This is done to ensure there is no regressions in your code. We cannot merge a PR if the tests fail so we will ask you to fix bugs before to review your code.
```
Comment (aledeg) [inline docs/en/developers/03_Running_tests.md]:
```suggestion
This is done to ensure your code does not introduce some kind of regression. We will not merge a PR if tests fail so we will ask you to fix any bugs before reviewing your code.
```

File: docs/fr/developers/03_Running_tests.md [hunk 2]
```diff
@@ -24,12 +24,10 @@ you can run the command directly with:
 NO_DOCKER=true make test
 ```
 
-## Travis
+## GitHub Actions for Continuous Integration
```
Comment (aledeg) [inline docs/fr/developers/03_Running_tests.md]:
```suggestion
## Intégration continue avec GitHub Actions
```

File: docs/fr/developers/03_Running_tests.md [hunk 3]
```diff
@@ -24,12 +24,10 @@ you can run the command directly with:
...
 
-Tests are automatically run when you open a pull request on GitHub. It is
-done with [Travis CI](https://travis-ci.org/FreshRSS/FreshRSS/). This is
-done to ensure there is no regressions in your code. We cannot merge a PR if
-the tests fail so we'll ask you to fix bugs before to review your code.
+Tests are automatically run when you open a pull request on GitHub.
```
Comment (aledeg) [inline docs/fr/developers/03_Running_tests.md]:
```suggestion
Les tests sont lancés automatiquement dès que vous ouvrez une « pull request » sur GitHub.
```

File: docs/fr/developers/03_Running_tests.md [hunk 4]
```diff
@@ -24,12 +24,10 @@ you can run the command directly with:
...
+They are done with [GitHub Actions](https://github.com/FreshRSS/FreshRSS/actions).
```
Comment (aledeg) [inline docs/fr/developers/03_Running_tests.md]:
```suggestion
Ceux-ci sont lancés grace aux « [GitHub Actions](https://github.com/FreshRSS/FreshRSS/actions) ».
```

File: docs/fr/developers/03_Running_tests.md [hunk 5]
```diff
@@ -24,12 +24,10 @@ you can run the command directly with:
...
+This is done to ensure there is no regressions in your code. We cannot merge a PR if the tests fail so we will ask you to fix bugs before to review your code.
```
Comment (aledeg) [inline docs/fr/developers/03_Running_tests.md]:
```suggestion
Cette action est nécessaire pour s'assurer qu'aucune régression ne soit introduite dans le code. Nous n'accepterons aucune PR si les tests ne sont pas valides, nous vous demanderons donc de corriger tout ce qui doit l'être avant de commencer à relire votre code.
```

File: docs/fr/developers/03_Running_tests.md [hunk 6]
```diff
@@ -24,12 +24,10 @@ you can run the command directly with:
...
 
-If you're interested in, you can take a look at [the configuration
-file](https://github.com/FreshRSS/FreshRSS/blob/edge/.travis.yml).
+If you are interested, you can take a look at [the configuration file](https://github.com/FreshRSS/FreshRSS/blob/edge/.github/workflows/tests.yml).
```
Comment (aledeg) [inline docs/fr/developers/03_Running_tests.md]:
```suggestion
Si cela vous intéresse, vous pouvez étudier [le fichier de configuration](https://github.com/FreshRSS/FreshRSS/blob/edge/.github/workflows/tests.yml).
```

**Review by aledeg — CHANGES_REQUESTED** [review]

File: README.md [hunk 7]
```diff
@@ -1,4 +1,3 @@
-[![Build Status][travis-badge]][travis-link]
```
Comment (Frenzie) [inline README.md:1]: You don't want a GHA badge? :-)

File: docs/en/developers/03_Running_tests.md [hunk 8]
```diff
@@ -27,8 +27,10 @@ make lint-fix # or, to fix the errors detected by the linter
 
 Similarly to PHPUnit, it downloads a [PHP\_CodeSniffer](https://github.com/squizlabs/PHP_CodeSniffer) binary (i.e. `phpcs` or `phpcbf` depending on the command) and verifies its checksum.
 
-## Travis
+## GitHub Actions for Continuous Integration
 
-Tests are automatically run when you open a pull request on GitHub. It is done with [Travis CI](https://travis-ci.org/FreshRSS/FreshRSS/). This is done to ensure there is no regressions in your code. We cannot merge a PR if the tests fail so we'll ask you to fix bugs before to review your code.
+Tests are automatically run when you open a pull request on GitHub.
+They are done with [GitHub Actions](https://github.com/FreshRSS/FreshRSS/actions).
```
Comment (Frenzie) [inline docs/en/developers/03_Running_tests.md]:
```suggestion
They are performed with [GitHub Actions](https://github.com/FreshRSS/FreshRSS/actions).
```

File: docs/en/developers/03_Running_tests.md (hunk 1 above)
Comment (Frenzie) [reply docs/en/developers/03_Running_tests.md]: (minor changes to @aledeg's suggestion)
```suggestion
This ensures your code will not introduce some kind of regression. We will not merge a PR if tests fail so we will ask you to fix any bugs before reviewing your code.
```

File: README.md (hunk 7 above)
Comment (Alkarex) [reply README.md:1]: Might be nice :-)

**Pull request merged**